### PR TITLE
Brian/add event skeleton

### PR DESCRIPTION
**Pull request by brianlevis**
It seemed like using a Navigator was a good option for the multi-step Add Event wizard we outlined in Figma, but the warning (see TODO) made me second-guess that.

@abhishyantkhare as someone who actually know how to use RN does this look reasonable or is Modal still better here?

### CODE REVIEW

File: imdown-mobile/login/login.tsx [hunk 1]
```diff
@@ -7,8 +7,9 @@ import { TouchableHighlight } from "react-native-gesture-handler";
 
 
 const Login = ({ navigation }) => {
-  const IOS_CLIENT_ID = '1097983281822-8qr8vltrud1hj3rfme2khn1lmbj2s522.apps.googleusercontent.com'
-  const LOGIN_SUCCESS = "success"
+  const IOS_CLIENT_ID = "1097983281822-8qr8vltrud1hj3rfme2khn1lmbj2s522.apps.googleusercontent.com";
+  const ANDROID_CLIENT_ID = "1097983281822-4b63n721lbqllpn7u4cvoqmh0rudquma.apps.googleusercontent.com";
```
Comment (abhishyantkhare) [inline imdown-mobile/login/login.tsx:11]: lol eventually we should probably move these to not be in code but all good for now

Comment (brianlevis) [reply imdown-mobile/login/login.tsx:11]: yeah I mean these params are only used for the expo mobile app anyway (and they are safe for public exposure)

**Comment by abhishyantkhare** [comment]
I think the issue is less so in passing data and more that we shouldn't pass methods in the route params, so we shouldn't pass in `addEvent` as a route param. Data itself is fine as long as its JSON serializable. 

IMO this is actually a short term problem because we'll eventually connect this to the backend, so when we add an event we can just POST it to the backend and the events page can just GET all the events from the backend. That way each page can be independent and we won't have to worry about how to pass data between pages (and if we want to do offline support we can utilize Redux which will solve the problem in a similar fashion).

So if we consider that the eventual goal is to hook this up to the backend then I think implementing this the way you've done it is fine, and will be a better solution in the future even if there's a short term problem now. I think we'd want to wrap the AddEvent in a navigation stack based on the designs so its fine.

**Comment by brianlevis** [comment]
> So if we consider that the eventual goal is to hook this up to the backend then I think implementing this the way you've done it is fine, and will be a better solution in the future even if there's a short term problem now. I think we'd want to wrap the AddEvent in a navigation stack based on the designs so its fine.

Ah, yeah I didn't even consider that

**Pull request merged**